src/core.rs

```diff
@@ -20,6 +21,7 @@ pub struct Core {
     //display: Display,
     colors: Colors,
     sorters: Vec<(SortOrder, sort::SortFn)>,
+    exit_code: ExitCode,
```

Comment:
Don't think we should initialize it here. We can just have a return from the recursive parse.